seq_tools/validation/c650_sm_in_bam_matches_metadata.py

```diff
@@ -91,7 +91,11 @@ def check(self):
         if bams and len(all_sms) != 1:
             raise Exception("No SM or more than one SM found in BAM(s): '%s'. Please see earlier check status "
                             "for details." % ', '.join(all_sms))
-
+        elif bams and list(all_sms)[0]=="":
+            self.status = 'WARNING'
+            message = "SM is empty. Validation status: WARNING"
```

Comment:
Minor comment, we can add a note into message: 
message = "SM is empty. Validation status: WARNING. NOTE that submitterSampleId in metadata JSON will be used in the header of ARGO uniformly aligned sequences."

Reply:
Added suggested change to warning:
https://github.com/icgc-argo/seq-tools/blob/73b9c6283a85e8ebb2a9ccce2d9cfdaec9125127/seq_tools/validation/c650_sm_in_bam_matches_metadata.py#L96